dao/src/main/java/se/fortnox/reactivewizard/db/transactions/DaoTransactionsFlux.java

```diff
@@ -1,6 +1,6 @@
 package se.fortnox.reactivewizard.db.transactions;
 
-import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
 
 public interface DaoTransactionsFlux {
```

Comment:
Shouldn't we change this name? `DaoTransactionsReactor` and then replace `DaoTransactions` once rxjava support is removed?

Reply:
If we are going to change it in the future, perhaps we can leave it like this and change it to the correct one once its name is going to be DaoTransactions?